### ISSUE TITLE
UNSTABLE inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench)

### ISSUE THREAD

**Issue by jeanschmidt** [pytorch/pytorch#104337]
This job is flaky, seems that there is a stochastic process during the model evaluations and it happens that sometimes models displays a lower accuracy even when changes are not related.

I am marking those tests are flaky, and we can then discuss if we can identify the source of stochastiness and maybe fix any RNG seed or maybe get rid of this test at all.

cc @ezyang @gchanan @zou3519 @seemethere @malfet @pytorch/pytorch-dev-infra @msaroufim @wconstab @bdhirsh @anijain2305

**Comment by ezyang** [comment]
This is an extremely important E2E job for inductor correctness and we need to resolve this.

**Comment by huydhn** [comment]
As I don't see the job failing in trunk anymore since last Friday, let's close this.  The last flaky failure was https://hud.pytorch.org/pytorch/pytorch/commit/36c4dad197343077acbf37b0fb9b4075f0010ccd

**Comment by huydhn** [comment]
This job is still pretty flaky on HUD https://hud.pytorch.org/hud/pytorch/pytorch/main/1?per_page=50&name_filter=nductor%20%2F%20cuda11.8-py3.10-gcc7-sm86%20%2F%20test%20(inductor_torchbench due to `vision_maskrcnn   IMPROVED: graph_breaks=51, expected=56` I think

**Comment by izaitsevfb** [comment]
We have two inductor torchbench tests failing since today:

https://www.torch-ci.com/hud/pytorch/pytorch/main/1?per_page=50&name_filter=nductor%20%2F%20cuda11.8-py3.10-gcc7-sm86%20%2F%20test%20(inductor_torchbenc

inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench_dynamic, 1, 1, linux.[g5](https://internalfb.com/intern/bunny/?q=g5).4xlarge.nvidia.gpu)
inductor / cuda11.8-py3.10-gcc7-sm86 / test (inductor_torchbench, 1, 1, linux.[g5](https://internalfb.com/intern/bunny/?q=g5).4xlarge.nvidia.gpu)

Error: 9 models have accuracy status regressed:
    Background_Matting LearningToPaint clip doctr_det_predictor doctr_reco_predictor drq sam soft_actor_critic yolov3


The root cause seem to be cv2 becoming incompatible, and it’s not caused by any particular commit:
```
File "/opt/conda/envs/py_3.10/lib/python3.10/site-packages/cv2/typing/__init__.py", line 69, in <module>
    NumPyArrayGeneric = numpy.ndarray[typing.Any, numpy.dtype[numpy.generic]]
TypeError: 'numpy._DTypeMeta' object is not subscriptable
```

**Comment by izaitsevfb** [comment]
Fixed in #106900
